src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -585,17 +553,14 @@
 
       async searchRecipients() :Promise<void> {
         this.partners = []
-        if(this.searchName == "" && !this.displayFavoritesOnly) {
-          this.searchRecipientsHistory()
-        } else {
-          var recipients
-          try {
-            recipients = await this.$lokapi.searchRecipients(this.searchName)
-          } catch (err) {
-            console.log('searchRecipients() Failed', err)
-          }
-          this.partners = this.displayFavoritesOnly ? returnFavoritesOnly(recipients) : recipients
+        var recipients
+        try {
+          recipients = await this.$lokapi.searchRecipients(this.searchName)
+          console.log(recipients)
```

Comment:
Trailing console.log ?

Reply:
@seegwen it's done, thank you your reviews

Reply:
@SeddikKadi many thanks. That's OK for me.

Now we need to fix the list (cf the other issue) which is critical for the first alpha release in my opinion.